<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>UserTaskUI</class>
 <widget class="QWidget" name="UserTaskUI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Select Users</string>
  </property>
  <property name="whatsThis" >
   <string>&lt;b>Select Users&lt;/b>&lt;br>
Double click on any user to add or remove them, or select the users, and click the arrow buttons.  You can filter the available user list by selecting a task type from the combo box.  You can edit what task types a user is associated with by clicking on the list on the right.</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>3</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="1" column="0" >
        <widget class="QListWidget" name="mAvailableList" />
       </item>
       <item row="1" column="1" >
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QPushButton" name="mAddUser" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>1</hsizetype>
             <vsizetype>3</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize" >
            <size>
             <width>35</width>
             <height>32767</height>
            </size>
           </property>
           <property name="text" >
            <string>>></string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mRemoveUser" >
           <property name="sizePolicy" >
            <sizepolicy>
             <hsizetype>1</hsizetype>
             <vsizetype>3</vsizetype>
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize" >
            <size>
             <width>35</width>
             <height>32767</height>
            </size>
           </property>
           <property name="text" >
            <string>&lt;&lt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0" >
        <widget class="QLabel" name="textLabel6" >
         <property name="text" >
          <string>Available Users:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" >
        <widget class="QListWidget" name="mSelectedList" />
       </item>
       <item row="0" column="2" >
        <widget class="QLabel" name="textLabel5" >
         <property name="text" >
          <string>Selected Users:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="mPixmapLabel" >
         <property name="scaledContents" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="mNameLabel" >
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="mRoleList" />
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="TaskTypeCombo" name="mRoleCombo" >
       <property name="minimumSize" >
        <size>
         <width>160</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>200</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mOKButton" >
       <property name="text" >
        <string>&amp;OK</string>
       </property>
       <property name="shortcut" >
        <string>Alt+O</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mCancelButton" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
       <property name="shortcut" >
        <string>Alt+C</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <widget class="QWidget" name="widget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>TaskTypeCombo</class>
   <extends></extends>
   <header>tasktypecombo.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mAddUser</tabstop>
  <tabstop>mRemoveUser</tabstop>
  <tabstop>mOKButton</tabstop>
  <tabstop>mCancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
